server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/comment/IssueCommentConverter.java

```diff
@@ -2,9 +2,11 @@
 
 import org.kohsuke.github.GHIssueComment;
 import org.springframework.lang.NonNull;
+import org.springframework.stereotype.Service;
 
 import de.tum.in.www1.hephaestus.codereview.base.BaseGitServiceEntityConverter;
 
+@Service
```

Comment:
I guess `@Component` for the converters is more fitting.

Reply:
You could also move the annotation to the `BaseGitServiceEntityConverter`

Reply:
> You could also move the annotation to the `BaseGitServiceEntityConverter`

It tried this, but fortunately it isn't enough for the compiler to auto-inject all subclasses. Changed it to `@Component` now though.